maven-plugin/src/main/java/org/openmrs/maven/plugins/Run.java

```diff
@@ -73,13 +81,113 @@ public void executeTask() throws MojoExecutionException, MojoFailureException {
 			}
 		}
 
+		try {
+			File tempFolder = createTempReactorProject(server);
+			cleanInstallModule(tempFolder.getPath());
+			deleteTempReactorProject(tempFolder);
```

Comment:
Probably this should be executed in a finally block to make sure we always clean up properly.